app/controllers/jobs_controller.rb

```diff
@@ -4,6 +4,7 @@ class JobsController < ApplicationController
 
 	def index
 		@jobs = Job.all.includes(:shipment, :client, shipment: :products, shipment: :shipment_products)
+		@user_suggestions = Job.search_on(current_user).tagged_with(current_user.tag_list, :any => true).order("RANDOM()").limit(10)
```

Comment:
What's the purpose of the `search_on` method for `Job`?

I have a feeling that this method shouldn't belong in the `Job` model - what do you think @rabingaire ?

Also, Let's make a method called `suggested_users` inside `User.rb`:

```
def suggested_users
  # implement here
end
```

Then we can call the method like this:

```current_user.suggested_users```

That way we are *telling* what we want to do, instead of `asking` ;) (Tell, don't ask) https://robots.thoughtbot.com/tell-dont-ask